src/Business Foundation/App/NoSeries/readme_refactoring.md

```diff
@@ -0,0 +1,133 @@
+# Refactoring to use the new No. Series module
+
+This document is created to give a brief explanation of how refactor your code to use the new No. Series module.
+Old refers to the current ways of using the No. Series implementation while New refers to using the newly created No. Series module.
+
+## Uptake examples
+
+### TryGetNextNo
+
+Old:
+```
+DocNo := NoSeriesMgt.TryGetNextNo(GenJnlBatch."No. Series", EndDateReq);
+```
+New:
+```
+DocNo := NoSeries.PeekNextNo(GenJnlBatch."No. Series", EndDateReq);
+```
+
+### GetNextNo with delayed modify
+
+Old:
+```
+if DocNo := NoSeriesMgt.GetNextNo(GenJnlBatch."No. Series", EndDateReq, false) then
+    NoSeriesMgt.SaveNoSeries();
+```
+New:
+You have two options here, either you can Peek the next No. and update or use batch
+```
+if DocNo := NoSeries.PeekNextNo(GenJnlBatch."No. Series", EndDateReq) then
+    NoSeries.GetNextNo(GenJnlBatch."No. Series", EndDateReq);
+```
+or
+```
+if DocNo := NoSeriesBatch.GetNextNo(GenJnlBatch."No. Series", EndDateReq) then
+    NoSeriesBatch.SaveState();
+```
+
+### InitSeries
+
+Now InitSeries is a very complex implementation. In most cases where InitSeries is used, we verify that the given No. is not set, which this example will also verify. If it is, you will need to verify whether manual Nos are allowed.
```

Comment:
Little more info would be nice here. Why is it a complex implementation? How do you verify how manual numbers are allowed? Do we need to abbreviate "No.," or can we write it as "number?"

```suggestion
Now, InitSeries is a complex implementation. In most cases that use InitSeries, we verify that the given No. isn't set, as shown in the following example. If it is, you'll need to verify whether manual Nos are allowed.
```